index.html

```diff
@@ -0,0 +1,137 @@
+            <li><a href="#" onclick="hideMenu()">Program</a></li>
+            <li><a href="#" onclick="hideMenu()">Join</a></li>
+            <li><a href="#" onclick="hideMenu()">Sponsor</a></li>
+            <li><a href="#" onclick="hideMenu()">News</a></li>
+          </ul>
+          <a href="" class="camp" onclick="hideMenu()">GDC Campaign</a>
+        </div>
+      </nav>
+      <div class="hero-text">
+        <div class="text">
+          <p>"Hello! Sharing world"</p>
+          <p class="title">GAME DEVELOPERS CONFERENCE SHOWCASE 2022</p>
+        </div>
+        <p class="sub-title"> A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in March, in USA.</p>
+        <div class="date-venue">
+          <p class="date">2022-03-21(MON) ~ 25(FRI)</p>
+          <p class="venue">@ Moscone Center San Francisco, CA USA</p>
+        </div>
+      </div>
+    </header>
+    <!-- Program section -->
+    <section class="program">
+      <h2 class="titles">Main Program</h2>
+      <div class="details">
+        <!-- program 1 -->
+        <div class="detail-col">
+          <img src="/assets/pro1.png" alt="detail1">
+          <h2>Lecture</h2>
+          <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </div>
+        <!-- program 2 -->
+        <div class="detail-col">
+          <img src="/assets/pro2.png" alt="detail2">
+          <h2>GDC Exhibition</h2>
+          <p>Appreciate various creations applying GDC license of artists, organized from GDC home office.</p>
+        </div>
+        <!-- program 3 -->
+        <div class="detail-col">
+          <img src="/assets/pro3.png" alt="detail3">
+          <h2>Forum</h2>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <!-- program 4 -->
+        <div class="detail-col">
+          <img src="/assets/pro4.png" alt="detail4">
+          <h2>Workshop</h2>
+          <p>Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <!-- program 5 -->
+        <div class="detail-col">
+          <img src="/assets/pro5.png" alt="detail5">
+          <h2>GDC Ignite</h2>
+          <p>Get opportunities to network with GDC affiliates around the world, also after the conference.</p>
+        </div>
+      </div>
+      <a href="#" class="desktop-text">see the whole program</a>
+      <a href="#" class="mobile-text">Join GDC 2022</a>
+    </section>
+    <!-- featured speakers section -->
+    <section class="speakers">
+      <h2 class="titles">Featured Speakers</h2>
+      <div class="speakers-list"></div>
+      <button type="button" class="btn-extend">
+        <span class="btn-text">More</span>
+        <img src="/assets/arrow_down.png" class="btn-pic" alt="down">
+      </button>
```

Comment:
Please for both the desktop and mobile version of the `featured speakers section` , you should add the top background image,as it should be looking at the design given. Please try to have a detailed perspective about your design to convince your client that you have an eyes for the job 😄 . Also you should take a closer look at your font for all paragrahs in your design, as it seems like you didnt use the right fon-type of `cococoose` and colors for the paragraphs, and they look different from your design

Design given :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/5f8eead8-ccee-4030-aec6-c6d35be92236)

Your design :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/30b621ab-bfb3-49a8-bdba-884f7be438db)